### ISSUE TITLE
Failed to Execute youtube-dl

### ISSUE THREAD

**Issue by levone1** [tramhao/termusic#66]
Using v0.6.13 on Sailfish OS. youtube-dl and ffmpeg both installed, and both commands run fine in terminal. When attempting to use "Download from YouTube" feature in Termusic, immediately fails with above error.
How to troubleshoot?
Thanks

**Comment by tramhao** [comment]
Yes, because the slow downloading speed of youtube-dl, I switchted to yt-dlp in certain version. Please install yt-dlp and it should be ok. Thanks.

**Comment by levone1** [comment]
> Yes, because the slow downloading speed of youtube-dl, I switchted to yt-dlp in certain version. Please install yt-dlp and it should be ok. Thanks.

yep - thanks
